generators/fastlane-setup/templates/deploy.sh

```diff
@@ -0,0 +1,121 @@
+#! /bin/bash
```

Comment:
Deployment should be handled by fastlane
I haven't updated the M33 code push standard but it should look like this
https://github.com/Minishlink/DailyScrum/blob/master/fastlane/Fastfile#L109